code/api/src/config/database.json

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
Was this file not needed?

Reply:
This is the file that includes my postgres username and password, we should all have unique ones, so I decided not include mine.